erpnext/selling/doctype/sales_order_item/sales_order_item.json

```diff
@@ -158,7 +158,8 @@
    "in_list_view": 1,
    "label": "Delivery Date",
    "no_copy": 1,
-   "print_hide": 1
+   "print_hide": 1,
```

Comment:
why print hide ?

Reply:
Didn't change/add the print_hide property. Current Bloomstack/erpnext develop branch already has it set.